doc/rtd/reference/yaml_examples/archive.rst

```diff
@@ -0,0 +1,20 @@
+.. _cce-archive:
```

Comment:
I would kill this page. We have this documented on the formats page and it's not cloud-config.

Reply:
Perfect, thanks!